packages/user/test/model.spec.ts

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import { signalToPromise } from '@jupyterlab/testutils';
+import { StateDB } from '@jupyterlab/statedb';
+
+import { User } from './../src/model';
+
+describe('user/model', () => {
+  describe('User', () => {
+    const stateDB = new StateDB();
+    let user: User;
+
+    it('Should emit user ready', async () => {
+      user = new User(stateDB);
```

Comment:
Looks like the `stateDB` doesn't need to be passed when instantiating a new `User` here in the tests?

(caught by the CodeQL check)